src/components/ui/DeveloperPreview.css

```diff
@@ -0,0 +1,3 @@
+.developer-preview__inline-right {
```

Comment:
Instead of defining custom class we could use built in PatternFly class `pf-u-text-align-right` (https://www.patternfly.org/v4/documentation/core/utilities/alignment#usage)

Reply:
done